internal/provider/resource_subaccount.go

```diff
@@ -361,3 +384,18 @@ func (rs *subaccountResource) Delete(ctx context.Context, req resource.DeleteReq
 func (rs *subaccountResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
 }
+
+func mapUsageToUsedForProduction(subaccountUsage string) string {
+	// The BTP CLI and CIS use different parameters for the subaccount usage
+	// To trigger the right usage creation in CREATE and avoid unwanted state changes in UPDATE  we must distinguish if and how to set the value
+	// Options: "" == ignored in request, "true" == boolean true in request, "false" == boolean false in request
+	switch subaccountUsage {
+	case "UNSET":
+		return "false"
```

Comment:
Is this required to be interpreted as false or could we also return "" in this case?

Reply:
Very good catch, That's indeed wrong: Unset must be empty string